src/domain/camera.py

```diff
@@ -172,3 +173,28 @@ def run(self):
             logger.error("Unable to create new thread for camera %s", self.id)
 
         return thread
+
+    def serialize(self):
+        """Method to serialize Camera object into file."""
+        data = dict(
```

Comment:
You can directly give back the dict. And you can use the dict literal {} for defining the dictionary.